src/pages/workspace/reimburse/WorkspaceRateAndUnitPage.js

```diff
@@ -39,82 +36,34 @@ const defaultProps = {
     ...policyDefaultProps,
 };
 
-class WorkspaceRateAndUnitPage extends React.Component {
-    constructor(props) {
-        super(props);
-        this.submit = this.submit.bind(this);
-        this.validate = this.validate.bind(this);
-
-        this.state = {
-            rate: 0,
-            unit: CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES,
-        };
-    }
-
-    componentDidMount() {
-        this.resetRateAndUnit();
-
-        if (lodashGet(this.props, 'policy.customUnits', []).length !== 0) {
+function WorkspaceRateAndUnitPage(props) {
+    const fetchData = useCallback(() => {
```

Comment:
This useCallback seems unnecessary. We only use this function in the useEffect hook and we can provide a dependency array there too. Also, should we be more specific and provide `props.policy` and `reimbursementAccount.isLoading` as the dependency?

Reply:
After removing the callback and putting it into the useEffect hook I had to leave the dependency list as is. `props` have to be passed anyways which makes adding the others redundant.